File: python/carcustomized.py
from car import Car

class CarCustomized(Car):
    seat_material       = []
    type_car_accepted   = []


    def __init__(self, id,number_plate, category, assembly_date, enrolled, brand, model, enrrollment_date, passengers, capacity, driver, seat_material, type_car_accepted):
        super.__init__(id,number_plate, category, assembly_date, enrolled, brand, model, enrrollment_date, passengers, capacity, driver)
        self.type_car_accepted = type_car_accepted
        self.seat_material = seat_material

File: js/CarCustomized.js
class CarCustomized extends Car
{
    constructor(number_plate, category, brand, model, driver, seat_material, type_car_accepted)
    {
        super(number_plate, category, brand, model, driver);
        this.seat_material=seat_material;
        this.type_car_accepted=type_car_accepted;
    }
}